.dockerignore

```diff
@@ -1 +0,0 @@
-.gitignore
```

Comment:
Yes, you are right.  This file should not have been changed.  I will restore back this.